test/ServerComparison.FunctionalTests/HelloWorldTest.cs

```diff
@@ -32,14 +31,26 @@ public Task HelloWorld_Windows_CLR(ServerType serverType, RuntimeFlavor runtimeF
         [ConditionalTheory]
         [OSSkipCondition(OperatingSystems.Linux)]
         [OSSkipCondition(OperatingSystems.MacOSX)]
-        [InlineData(ServerType.IISExpress, RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Portable, Skip = "Tests disabled on x86 because of https://github.com/aspnet/Hosting/issues/601")]
         [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Portable)]
         [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
-        public Task HelloWorld_Windows(ServerType serverType, RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, ApplicationType applicationType)
+        public Task HelloWorld_HttpSys(ServerType serverType, RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, ApplicationType applicationType)
         {
             return HelloWorld(serverType, runtimeFlavor, architecture, applicationType);
         }
 
+        [ConditionalTheory]
```

Comment:
[FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)] to prevent running the tests twice.

Reply:
Why not change test project to only have one TFM?

Reply:
Because there are tests that cover other servers like Kestrel

Reply:
Why would I block CoreClr here for this test?

Reply:
Remember that these tests all run in a different process and choose their own clr. The TFM only controls the test process. There's no reason to run these tests twice with different tfm's, it doesn't affect the other process.

Reply:
I thought the point of running these tests with multiple tfms was for client behavior verification for netcoreapp2.1 and net461? We special case some clients due to incorrect behavior.

Reply:
We don't care about the client behavior except where it makes it easier or harder to verify the server behavior. E.g. the NTLM tests are run on net4x because the netcoreapp2.0 client has a behavior that makes it difficult to test the server. We should try the new netcoreapp2.1 client behavior and see if we can consolidate the test project to that TFM.